Packs/Slack/Scripts/SlackBlockBuilder/SlackBlockBuilder.py

```diff
@@ -157,10 +157,18 @@ def _parse_url_blocks(self):
         Slack provides a tool located at https://app.slack.com/block-kit-builder. When you are done, you can copy and
         paste the URL into the blocks_url argument for this automation. The URL is then decoded to provide the blocks.
         """
-        url_encoded_blocks: str = self.url.split('#')[1]
-        url_decoded_blocks: str = urllib.parse.unquote(url_encoded_blocks)
-        parsed_blocks: Any = json.loads(url_decoded_blocks)
-        self.blocks_dict = parsed_blocks.get('blocks', [{}])
+        try:
+            if match_url_encoded_blocks := re.search(r"#(.+)", self.url):
```

Comment:
maybe add in the description of this function why the regex needed and been used